packages/xstate-react/README.md

```diff
@@ -141,7 +141,7 @@ const [state, send] = useActor(customActor, (actor) => {
 
 ### `useInterpret(machine, options?, observer?)`
 
-A React hook that returns the `service` created from the `machine` with the `options`, if specified. It also sets up a subscription to the `service` with the `observer`, if provided.
+A React hook that returns the `service` created from the `machine` with the `options`, if specified. It also sets up a subscription to the `service` with the `observer`, if provided. It starts the service and runs it for the lifetime of the component. Similar to using `useMachine`, but with an observer.
```

Comment:
We could somehow mention here that the main difference here is that `useMachine` pumps all changes to React's state (using the observer, since it's currently built on top of the `useInterpret`). While `useInterpret` gives you a more fine-grained control over this

Reply:
```suggestion
A React hook that returns the `service` created from the `machine` with the `options`, if specified. It starts the service and runs it for the lifetime of the component.

It also sets up a subscription to the `service` with the `observer`, if provided.

Using `useInterpret` is useful when you want to minimise re-renders. Where `useMachine` flushes each update from the machine to the component, but `useInterpret` returns a static reference which never causes a render.

To use a piece of state from the service inside a render, you'll need to use `useSelector` to subscribe to it.
```

Reply:
@mattpocock should it be "never causes a render" or "never causes a re-render" ?